app.rb

```diff
@@ -0,0 +1,171 @@
+require 'person'
+require 'book'
+require 'student'
+require 'classroom'
+require 'rental'
+require 'teacher'
```

Comment:
- [ ] I could not start your app because of import errors. You should use `require_relative` when importing local files. Based on convention. `require` is meant for external files and packages like gems.

  ![image](https://github.com/mohamedgamaleldin999999/School-library/assets/36542986/10d5e8db-6ca6-4771-9529-1ade6a7bbad2)